### PR TITLE
Blacksmith updates

### PR DESCRIPTION
**Pull request by TheRedPanda17**

### CODE REVIEW

File: myning/migrations/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from . import (
+    black_smith_upgrade,
```
Comment (rj-lee) [inline myning/migrations/__init__.py:2]:
```suggestion
    blacksmith_upgrade,
```

File: myning/chapters/enter_blacksmith.py [hunk 2]
```diff
@@ -118,4 +120,4 @@ def generate_items(classes: List["BlacksmithItem"]):
 
 
 def smith_cost(level):
-    return utils.fibonacci(level + 2) * 100
+    return utils.fibonacci(level + 4) * 100
```
Comment (rj-lee) [inline myning/chapters/enter_blacksmith.py:123]: lol gg early game

File: myning/migrations/black_smith_upgrade.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+from myning.utils.file_manager import FileManager
+
+
+def run():
+    with open(".data/player.json") as f:
+        player = Player()
+
+        if player.has_upgrade("buy_full_set"):
+            upgrade = UPGRADES["buy_full_set"]
+            upgrade.level = 5
```
Comment (rj-lee) [inline myning/migrations/black_smith_upgrade.py:14]: this is Jedi and not Blademaster?

File: myning/migrations/__init__.py (hunk 1 above)
Comment (TheRedPanda17) [reply myning/migrations/__init__.py:2]: Yup I'll fix it.

File: myning/chapters/enter_blacksmith.py (hunk 2 above)
Comment (TheRedPanda17) [reply myning/chapters/enter_blacksmith.py:123]: It's not too much more expensive

File: myning/migrations/black_smith_upgrade.py (hunk 3 above)
Comment (TheRedPanda17) [reply myning/migrations/black_smith_upgrade.py:14]: Yeah. If you bought it before, you spent 100k to do so. 100k gets you to around level 5 I think.

Comment (TheRedPanda17) [reply myning/migrations/black_smith_upgrade.py:14]: Maybe should be level 6 actually

Comment (rj-lee) [reply myning/migrations/black_smith_upgrade.py:14]: level 5 is a little less in total money. and before, your upgrade was for the blademaster set so 6 makes sense to "grandfather" the legacy upgrade. although I don't think anyone has this right now but you

**Pull request merged**